### PR TITLE
Deprecate repository

### LINKED ISSUE #123: Add cli command to download files specified in manifest file constructed by the data browser
The Data Browser facet search will construct a manifest file specifying UUIDs and timestamps for files the user wants to download.
We need to provide a "hca download <manifest_file>" cli command to do the actual download (including retries).

### LINKED ISSUE #456: DSS GET /bundles/all does not iterate correctly
`import hca; list(hca.dss.DSSClient().get_bundles_all.iterate(replica="aws"))` produces nothing (after fetching all of the results). I believe this has to do with iteration keys being hardcoded in https://github.com/HumanCellAtlas/dcp-cli/blob/4d844f3c3a299162c68e25e9ffc6ffe7e8bf7ce8/hca/util/__init__.py#L183-L199.

* The short-term workaround is to hardcode another special case for this route there.

* The long-term workaround is to include pagination keys in the swagger spec for the method.

Also, `hca dss get-bundles-all --replica aws` only returns the first page.

### PR DESCRIPTION
**Pull request by amarjandu**
Add deprecation within README.md
Remove uploading of .whl to pypi to force building from source
Does not allow installation via `pip install hca` see warning attached
Does allow installation via `pip install hca==7.0.0`

Screen Shot 2021-03-23 at 10 24 21 AM

### CODE REVIEW

**Comment by amarjandu** [comment]
I performed the deprecation testing on https://pypi.org/project/dbio-cli/. version 1.0.0 is a known good build, version 1.5.5 is the version that fails when you try to install.

File: setup.py [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 import os
 from setuptools import setup, find_packages
+import sys
+
+if not any(arg in sys.argv for arg in ('sdist', 'bdist_wheel', '--name', '--version')):
```
Comment (danielsotirhos) [inline setup.py:7]:
```suggestion
if set(sys.argv).isdisjoint({'sdist', 'bdist_wheel', '--name', '--version'})
```
This might read a little easier.

**Review by danielsotirhos — APPROVED** [review]
Should `databiosphere/azul#2897` be in the title? Seems like it should be a link in a comment instead.

Approved.

```
(.venv) daniel@Crispin ~/dcp-cli $ pip install git+https://github.com/HumanCellAtlas/dcp-cli.git@issues/amar/2897-RIP-cli
Collecting git+https://github.com/HumanCellAtlas/dcp-cli.git@issues/amar/2897-RIP-cli
  Cloning https://github.com/HumanCellAtlas/dcp-cli.git (to revision issues/amar/2897-RIP-cli) to /private/var/folders/8s/9dwlx2rj1n39n7mdk610dhj40000gn/T/pip-req-build-3qrgv4vs
  Running command git clone -q https://github.com/HumanCellAtlas/dcp-cli.git /private/var/folders/8s/9dwlx2rj1n39n7mdk610dhj40000gn/T/pip-req-build-3qrgv4vs
  Running command git checkout -b issues/amar/2897-RIP-cli --track origin/issues/amar/2897-RIP-cli
  Switched to a new branch 'issues/amar/2897-RIP-cli'
  Branch 'issues/amar/2897-RIP-cli' set up to track remote branch 'issues/amar/2897-RIP-cli' from 'origin'.
  Running command git submodule update --init --recursive -q
    ERROR: Command errored out with exit status 1:
     command: /Users/daniel/dcp-cli/.venv/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/8s/9dwlx2rj1n39n7mdk610dhj40000gn/T/pip-req-build-3qrgv4vs/setup.py'"'"'; __file__='"'"'/private/var/folders/8s/9dwlx2rj1n39n7mdk610dhj40000gn/T/pip-req-build-3qrgv4vs/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base pip-egg-info
         cwd: /private/var/folders/8s/9dwlx2rj1n39n7mdk610dhj40000gn/T/pip-req-build-3qrgv4vs/
    Complete output (3 lines):

    The HCA module is deprecated, for assistance with accessing data please refer to the data-browser quick start guide at https://data.humancellatlas.org/guides/quick-start-guide

    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
WARNING: You are using pip version 19.2.3, however version 21.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```

File: setup.py (hunk 1 above)
Comment (amarjandu) [reply setup.py:7]: Thanks! I'll implement this.

**Comment by amarjandu** [comment]
@danielsotirhos having the issue link the in PR title is wrong (I did not change it from the default oops), I wanted the link to be only within the commit message. 



I'm all for removing it from the commit message, but it does seem useful to have. 

I'll leave it within the commit message for now, and @hannes-ucsc so he can 👀

File: README.rst [hunk 2]
```diff
@@ -1,3 +1,7 @@
+DEPRECATED
+==========
+This repository is deprecated, for assistance with accessing data please refer to the `data-browser quick start guide. <https://data.humancellatlas.org/guides/quick-start-guide>`_
```
Comment (hannes-ucsc) [inline README.rst:3]:
```suggestion
This repository has been deprecated. For information about accessing Human Cell Atlas data please refer to the `data-browser quick start guide. <https://data.humancellatlas.org/guides/quick-start-guide>`_
```

File: setup.py [hunk 3]
```diff
@@ -2,6 +2,13 @@
 
 import os
 from setuptools import setup, find_packages
+import sys
+
+if set(sys.argv).isdisjoint({'sdist', 'bdist_wheel', '--name', '--version'}):
+    sys.exit('\nThe HCA module is deprecated, for assistance with accessing data '
```
Comment (hannes-ucsc) [inline setup.py:8]: Like above.

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]

**Pull request merged**